tests/integration/rsi/test_rsi_change_answers.py

```diff
@@ -0,0 +1,168 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class RsiHappyPath(IntegrationTestCase):
```

Comment:
Our class names for Tests usually start with Test and should probably be more descriptive e.g. `TestRsiSubmissionData`